app/code/core/Mage/Checkout/Helper/Data.php

```diff
@@ -230,7 +230,11 @@ public function sendPaymentFailedEmail($checkout, $message, $checkoutType = 'one
         $total = $checkout->getStoreCurrencyCode() . ' ' . $checkout->getGrandTotal();
 
         foreach ($sendTo as $recipient) {
-            $mailTemplate->setDesignConfig(['area' => 'frontend', 'store' => $checkout->getStoreId()])
+            $mailTemplate
+                ->setDesignConfig([
+                    'area' => Mage_Core_Model_App_Area::AREA_FRONTEND,
```

Comment:
```suggestion
                    'area'  => Mage_Core_Model_App_Area::AREA_FRONTEND,
```

Reply:
Ah yes I wasn't used to the "align the =>" form. Sorry for that 😅.